packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -463,6 +475,54 @@ class InspectorRowContent extends StatelessWidget {
     }
 
     final node = row.node;
+    final List<Widget> children = [
+      node.showExpandCollapse ?
+      InkWell(
+        onTap: onToggle,
+        child: RotationTransition(
+          turns: expandAnimation,
+          child: Icon(
+            Icons.expand_more,
+            size: 16.0,
+          ),
+        ),
+      ) : const SizedBox(width: 16.0, height: 16.0),
+      DecoratedBox(
+        decoration: BoxDecoration(
+          color: backgroundColor,
+        ),
+        child: InkWell(
+          onTap: () {
+            controller.onSelectRow(row);
+          },
+          child: Container(
+            height: rowHeight,
+            padding: const EdgeInsets.symmetric(horizontal: 4.0),
+            child: DiagnosticsNodeDescription(
+              node.diagnostic,
+            ),),
+        ),
+      ),
+    ];
+
+    if (controller.isExperimentalStoryOfLayoutEnabled) {
+      if (node.diagnostic?.isFlex == true)
+        children.add(
+          Container(
+            margin: const EdgeInsets.only(left: 4.0),
+            child: InkWell(
+              child: Icon(Icons.info, size: 16.0),
+              onTap: () {
+                showDialog(
```

Comment:
we should be using the tab not a dialog.

Reply:
is it okay to keep the button? I was thinking to remove the button and use similar approach with the Details tree (listening to select/focused node on the Summary tree) and display the layout information only when the layout tab is the active one